modules/ocf_mirrors/manifests/init.pp

```diff
@@ -156,4 +162,12 @@
     ssl_cert  => "/etc/ssl/private/${::fqdn}.crt",
     ssl_chain => '/etc/ssl/certs/incommon-intermediate.crt',
   }
+
+  cron {
+    'mirrors-stats':
+      command => '/usr/local/sbin/record-mirrors-stats',
```

Comment:
seems to work fine for the last two weeks I let it run, but yeah I need to make it +x